vdm-io/Joomla-Component-Builder
# Added CodeMirror to code areas all accross JCB

Pull Request for Issue gh-305 .

### Summary of Changes
Changed all/most textarea fields to editor fields with codemirror.

### Testing Instructions
Open all areas that has custom code areas and check if the values are editable and gets saved correctly.

### Expected result
That this transition from textarea fields to editor will improve JCB custom scripting.

### Actual result
New feature

### Documentation Changes Required
Well now we should have better control over the custom code, and so it should make things easier. We may need to make a tutorial to help those who do not know how to customize codemirror.
